### PR TITLE
#253 update dependencies

### LINKED ISSUE #253: Update dependencies
Placeholder issue for updating `npm` + `nrwl` dependencies.

## Expected Behaviour

All packages to latest version.

## Current Behaviour

Some out-of-date packages.

## Possible Solution

`nx migrate @nrwl/workspace`.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #253.

Branches off #255.

## Description

Bumps `nx` + `npm` dependencies.

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [ ] Tests OK.

### CODE REVIEW

File: unified-codes/package.json [hunk 1]
```diff
@@ -31,20 +31,20 @@
   },
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.2.1",
-    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@apollo/client": "^3.2.4",
```
Comment (katherine-sussol) [inline unified-codes/package.json:34]: Wondering if we should look at removing this one now that we aren't using it?

**Review by katherine-sussol — APPROVED** [review]
Looks good to me

File: unified-codes/package.json (hunk 1 above)
Comment (wlthomson) [reply unified-codes/package.json:34]: Good idea. I wonder if there is a tool for trimming unused deps?

Comment (wlthomson) [reply unified-codes/package.json:34]: Update: [woop woop!](https://github.com/depcheck/depcheck) (note: seems to not work v. well for `devDependencies`, so be careful).

Comment (wlthomson) [reply unified-codes/package.json]: 6aa82b1c4521dac3eeede8c3f1225a496c367ffa. Also looks like some deps were missing from the `package.json`, possibly automatically installed by other packages... but acb6918013b1bcfa7a878552665688736b618190 just to be safe.

safe

**Comment by mark-prins** [comment]
maybe things have shifted already??
a few things have bumped over the weekend - might be time to update again?

There's also a couple of high vulnerabilities - last time I bumped versions it was a little too specific to fix them. only one is from an outdated package, so they'd have to be manual fixes which I'd rather avoid

Screen Shot 2020-10-27 at 9 17 50 AM

**Comment by wlthomson** [comment]
@mark-prins Nice spot!

One of the vulnerabiltiys is a regex DOS 🤣.

Updated everything, but looks like some vulnerabilties need manual review (probably nothing major, raise separate issues for these, if needed?).

fa77ef1a.

**Comment by mark-prins** [comment]
> One of the vulnerabiltiys is a regex DOS 🤣.

no way! I take it all back!!

**Comment by mark-prins** [comment]
yup. vulnerabilities are a problem for the future, if at all.. it get's too messy trying to patch versions of dependencies several layers deep

**Pull request merged**